packages/web/tsconfig.json

```diff
@@ -17,7 +17,6 @@
     "isolatedModules": true,
     "jsx": "preserve"
   },
-  "references": [{ "path": "../extractor" }],
```

Comment:
Does this change indicate you rather want to use the published version of the extractor for the web package instead of the local source?